src/utils/constants.js

```diff
@@ -16,3 +16,6 @@ export const SEARCH_STORAGE_REFERRAL_DATA_REFERRING_USER =
   'search.referralData.referringUser'
 export const SEARCH_STORAGE_REFERRAL_DATA_REFERRING_CHANNEL =
   'search.referralData.referringChannel'
+export const STORAGE_NEW_USER_CAUSE_ID = 'tab.newUser.causeId'
+export const STORAGE_CATS_CAUSE_ID = '6ce5ad8e-7dd4-4de5-ba4f-13868e7d212z'
```

Comment:
The cause ID for cats shouldn't be the same as the charity ID because they're different entities. Here's a new one:
`80511947-ee5e-4dcf-ad4f-ace552155477`

Reply:
@kmjennison jed made this cause id for cats in parallel with the one you provided, but his is pinned in slack.  leaving it here for clarity
7f8476b9-f83f-47ac-8173-4a1c2ec3dc29